docs/first-steps.rst

```diff
@@ -19,14 +19,17 @@ You can choose your backend by setting the environment variable ``GEOMSTATS_BACK
 
 .. code-block:: bash
 
-    export GEOMSTATS_BACKEND=numpy
+    export GEOMSTATS_BACKEND=pytorch
 
 **FIRST EXAMPLES**
 
-We illustrate here the use of `geomstats` to generalize learning
-algorithms to Riemannian manifolds.
+To use `geomstats` for learning
+algorithms to Riemannian manifolds, you need to follow three steps:
+- instantiate the manifold of interest,
+- instantiate the learning algorithm of interest,
+- run the algorithm.
```

Comment:
maybe mention that the data should be in a numpy array/ tensor ?

Reply:
done.